documentation/en/api-v1-unstable-methods.md

```diff
@@ -3383,6 +3387,61 @@ Response:
 }
 ```
 
+## Get
+
+
+### GetActorEvents
+Actor events
+
+
+Perms: read
+
+Inputs:
+```json
+[
+  {
+    "address": [
+      "f01234"
+    ],
+    "fields": {
+      "abc": [
+        {
+          "codec": 81,
+          "value": "ZGF0YQ=="
+        }
+      ]
+    },
+    "minEpoch": 2301220,
+    "maxEpoch": 2301220
+  }
+]
+```
+
+Response:
+```json
+[
+  {
+    "Entries": [
+      {
+        "Flags": 7,
+        "Key": "string value",
+        "Codec": 42,
+        "Value": "Ynl0ZSBhcnJheQ=="
+      }
+    ],
+    "EmitterAddr": "f01234",
+    "Reverted": true,
```

Comment:
Maybe this is already documented elsewhere, but what does it mean when an event is reverted?

It would be nice to add a bit more documentation for the other event fields like Height, TipSetKey and MsgCid too. Copying the comments you have in your Go source code is a good start.

Reply:
@bajtos 

Given how Filecoin tipsets work, `reverted` basically means that the transition implied by the event no longer holds true as the tipset/block that caused the event has been reverted i.e. transactions in the block that were applied have been reverted and the state transition caused by those transactions no longer hold true.

Reply:
Makes sense, thanks for the explanation 👍🏻 

Would you know where to find more documentation about when a tipset/block can get reverted?